src/style/CampaignPage.less

```diff
@@ -304,30 +304,98 @@ body {
 	}
 
 	.impact-hub-splash {
+		@media (max-height: 680px) {
+			height: 105vh; // short screens hid cta button
+		}
+		.splash-thanks {
+			text-shadow: none;
+			letter-spacing: 1px;
+			font-size: 1.3rem;
+			font-weight: 700;
```

Comment:
We try to avoid local font css -- as it leads to inconsistent font use. Would one of the "standard" set of h1, h2, h3 fit here? If not -- just leave a short comment.

Note: when a designer gives you a font-size that doesn't quite match elsewhere in the app -- sometimes that's intentional and should be respected -- and sometimes its accidental because they were focused on one thing.